transport/grpc/handler.go

```diff
@@ -0,0 +1,154 @@
+package grpc
+
+import (
+	"bytes"
+	"time"
+
+	"github.com/yarpc/yarpc-go/transport"
+	"github.com/yarpc/yarpc-go/transport/internal"
+
+	"strings"
+
+	"fmt"
+
+	"io"
+
+	"errors"
+
+	"net/url"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	gtransport "google.golang.org/grpc/transport"
+)
+
+var grpcOptions transport.Options
```

Comment:
Hmm, it looks like the alternative is inlining "transport.Options{}' into the function call (Which doesn't convey as much information imo).  This is also similar to how we do it in the http handler.